packages/integrations/solid/package.json

```diff
@@ -40,10 +40,10 @@
   "devDependencies": {
     "astro": "workspace:*",
     "astro-scripts": "workspace:*",
-    "solid-js": "^1.8.5"
+    "solid-js": "^1.8.7"
   },
   "peerDependencies": {
-    "solid-js": "^1.4.3"
+    "solid-js": "^1.8.7"
```

Comment:
Changing the peer dep range is a breaking change. I don't think we need to update the dependencies manually ourselves, users should update themselves if they want to use the latest version of solid?